RAKE/RAKE.py

```diff
@@ -27,16 +27,42 @@ def FoxStopList():
     return FoxStopList.words()
 
 
-def load_stop_words(stop_word_file):
+def MySQLStopList():
+    from .stoplists import MySQLStopList
+    return MySQLStopList.words()
+
+
+def NLTKStopList():
+    from .stoplists import NLTKStopList
+    return NLTKStopList.words()
+
+
+def load_stop_words(stop_word_file, divide):
+    """
+    Utility function to load stop words from a file and return as a list of words
+    @param stop_word_file Path and file name of a file containing stop words.
+    @return list A list of stop words.
+    """
+    stop_words = []
+    for line in open(stop_word_file):
+        if divide:
+            for word in line.split():  # in case more than one per line
+                stop_words.append(word)
+    return stop_words
+
+def load_stop_words_delimiter(stop_word_file, delimiter):
     """
     Utility function to load stop words from a file and return as a list of words
     @param stop_word_file Path and file name of a file containing stop words.
     @return list A list of stop words.
     """
     stop_words = []
     for line in open(stop_word_file):
-        for word in line.split():  # in case more than one per line
-            stop_words.append(word)
+        for word in line.split(delimiter):  # in case more than one per line
```

Comment:
I wonder if this might be a good use case for [re.split](https://docs.python.org/3.5/library/re.html#re.split), actually. That way users could supply a regex string that matches the delimiter as well as spaces, which would provide the same functionality as the divide function above (and provide a lot more flexibility in general).